### PR TITLE
literature_folders_markus

### PR DESCRIPTION
**Pull request by markus-chem**

### CODE REVIEW

**Comment by saraedum** [comment]
Build is failing due to https://github.com/echemdb/svgdigitizer/pull/96.

File: literature/endo_1999_in-situ_19/endo_1999_in-situ_19.bib [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@article{endo_1999_in-situ_19,
+  title={In-situ X-ray absorption studies of bromine on the Ag (100) electrode},
```
Comment (DunklesArchipel) [inline literature/endo_1999_in-situ_19/endo_1999_in-situ_19.bib:2]:
```suggestion
  title={In-situ X-ray absorption studies of bromine on the Ag(100) electrode},
```

File: literature/gomez-marin_2012_surface_558/gomez-marin_2012_surface_558.bib [hunk 2]
```diff
@@ -0,0 +1,9 @@
+@article{gomez-marin_2012_surface_558,
+  title={Pt (1 1 1) surface disorder kinetics in perchloric acid solutions and the influence of specific anion adsorption},
```
Comment (DunklesArchipel) [inline literature/gomez-marin_2012_surface_558/gomez-marin_2012_surface_558.bib:2]:
```suggestion
  title={Pt (111) surface disorder kinetics in perchloric acid solutions and the influence of specific anion adsorption},
```

**Review by DunklesArchipel — APPROVED** [review]
Looks fine.
I removed some typos in the title field.

Was it intentional that you upload two literature folders?

**Comment by markus-chem** [comment]
> Looks fine. I removed some typos in the title field.
> 
> Was it intentional that you upload two literature folders?

Es folgen noch ca 20 solcher Ordner. Jetzt werde ich aber versuchen einen PR pro Ordner zu machen und nicht zwei in einem.

**Comment by markus-chem** [comment]
Jetzt sind doch alle 20 Ordner in eienm PR gelandet. Hätte ich dann für jeden commit einen eigenen branch machen sollen?

**Comment by markus-chem** [comment]
Ich kann es gerne nochmal anders machen und wir schließen den PR - wenn das für euch leichter zum Reviewen ist.

**Comment by saraedum** [comment]
PRs should always be as small as possible I think so splitting this up is probably better. If it's too much work, we can probably also review this patch bomb ;)

**Comment by saraedum** [comment]
Btw., it would be great if you could try to use more informative log messages, see https://chris.beams.io/posts/git-commit/.

That's not terribly important and your log messages are also quite ok already but maybe it's a good habit to establish :)

**Comment by markus-chem** [comment]
Well, patch bomb :D
OK, so we close this PR, I do one branch per folder and then one PR per branch?
And good point with the log messages, thanks. I thought using this bibliographic identifier would be ok.

**Comment by markus-chem** [comment]
Just let me know what to do and I'll try. Good thing is that git tracks what (mistakes) I do.

**Comment by saraedum** [comment]
>  I do one branch per folder and then one PR per branch?

Sounds great to me.

> I thought using this bibliographic identifier would be ok.

Yes, it's quite reasonable. But maybe your future self will be happy to read an actual sentence when looking at the commit history of this project in a couple of years. Anyway, I mostly think this is about developing a good habit as you are just getting started with git.

**Pull request closed without merging**